wandb/sdk/launch/docker.py

```diff
@@ -232,14 +263,21 @@ def _get_docker_image_uri(name: str, work_dir: str) -> str:
     return name + version_string
 
 
-def _create_docker_build_ctx(work_dir: str, dockerfile_contents: str) -> str:
+def _create_docker_build_ctx(
+    work_dir: str, dockerfile_contents: str, config_path: str
+) -> str:
     """
     Creates build context tarfile containing Dockerfile and project code, returning path to tarfile
     """
     directory = tempfile.mkdtemp()
     try:
         dst_path = os.path.join(directory, "wandb-project-contents")
         shutil.copytree(src=work_dir, dst=dst_path)
+        if os.path.exists(config_path):
+            shutil.copyfile(
```

Comment:
what is this for just curious, didn't realize we had another intermediate copy going on before the docker image

Reply:
This is the build context path, docker only allows you to copy files from your build context path into the docker container. We could just store the config directly in the LaunchProject, in fact I think thats better. So I'll do that, and just write it to the build context path here